ajax/libs/antimoderate/package.json

```diff
@@ -16,15 +16,13 @@
       ]
     }
   ],
-  "license": {
-    "type": "See README.md",
-    "url": "https://github.com/whackashoe/antimoderate#license"
-  },
+  "license": "MIT",
   "author": {
     "name": "Jett LaRue",
     "url": "https://jettlarue.com"
   },
   "keywords": [
-    "antimoderate"
+    "antimoderate",
+    "image"
```

Comment:
Maybe add some more like `progressive`, `loading`?

Reply:
PS: You can also contribute this back to its own bower.json in its repository, or even help it create a package.json if its author accepts.